ui/blocks/BlocksContent.tsx

```diff
@@ -23,9 +23,11 @@ const TABS_HEIGHT = 88;
 interface Props {
   type?: BlockType;
   query: QueryWithPagesResult<'blocks'>;
+  showSocketInfo?: boolean;
```

Comment:
```suggestion
  enableSocket?: boolean;
```

since it is not only responsible for showing the info but also allowing to get data from the socket.